src/series.py

```diff
@@ -0,0 +1,44 @@
+"""This module calculates Fibonacci and Lucas sequences."""
+
+
+def fibonacci(n):
+    """Return the nth number of a Fibonacci sequence."""
+    series = []
+    for i in range(n):
+        if i == 0 or i == 1:
+            series.append(i)
+        else:
+            next_num = (series[i - 2]) + (series[i - 1])
```

Comment:
In Python, you don't need to wrap these "getitem" statements in parentheses; it just adds more characters, making your code hard to read.